pkg/manifests/common.go

```diff
@@ -52,7 +52,7 @@ func GetOwnerRefs(owner client.Object, controller bool) []metav1.OwnerReference
 	}}
 }
 
-func namespace(conf *config.Config) *corev1.Namespace {
+func Namespace(conf *config.Config) *corev1.Namespace {
```

Comment:
did this need to be made public? (don't see any out of package references)

Reply:
we use these in the Webhook package now so we can attach the ownership references to the webhook.